packages/lab/src/__tests__/__e2e__/slider/RangeSlider.cy.tsx

```diff
@@ -0,0 +1,286 @@
+import * as rangeSliderStories from "@stories/slider/range-slider.stories";
+import { composeStories } from "@storybook/react";
+
+const composedStories = composeStories(rangeSliderStories);
+
+const { Default } = composedStories;
+
+describe("Given a Range Slider", () => {
+  it("should render with default props", () => {
+    cy.mount(<Default />);
+
+    cy.findAllByRole("slider").should("exist");
+    cy.findAllByRole("slider").eq(0).should("have.value", "0");
+    cy.findAllByRole("slider").eq(1).should("have.value", "1");
+  });
+
+  it("should trigger onChange when clicked on the track", () => {
+    const changeSpy = cy.stub().as("changeSpy");
+    cy.mount(<Default style={{ width: "400px" }} onChange={changeSpy} />);
+    cy.get(".saltSliderTrack-rail").trigger("pointerdown", {
+      button: 0,
+      clientX: 750,
+      clientY: 50,
+    });
+    cy.get("@changeSpy").should("have.callCount", 1);
+    cy.get("@changeSpy").should(
+      "have.been.calledWith",
+      Cypress.sinon.match.any,
+      Cypress.sinon.match.array.deepEquals([0, 7]),
+    );
+    cy.get(".saltSliderTrack-rail").trigger("pointerup");
+  });
+
+  it("should set thumb positions correctly when clicked on the track", () => {
```

Comment:
[non-blocking] is there anything to test to determine which thumb should be moved